cmake/pkgconfig.in

```diff
@@ -6,6 +6,6 @@ Name: libOSRM
 Description: Project OSRM library
 Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
-Libs: -L${libdir} -lOSRM
+Libs: -L${libdir} -losrm
```

Comment:
There are still libs missing, at least pthread and rt; take a look at my old Gist on how to compile the simpleclient, and why pkg-config was not sufficient: https://gist.github.com/daniel-j-h/53499ae4534920312992